rocketpy/Environment.py

```diff
@@ -394,6 +404,11 @@ def setDate(self, date):
         # Store date
         self.date = datetime(*date)
 
+        if time_zone != None:
+            self.time_zone = time_zone
+            tz = pytz.timezone(self.time_zone)
+            self.local_date = self.date.replace(tzinfo=pytz.UTC).astimezone(tz)
+
```

Comment:
I am not sure that this is working as I expected. I expected that the user inputs the time in the local time zone, specifies which time zone is he using and the RocketPy internally converts this time to UTC in order to work properly. 

These lines seem to be doing the oposite. The user needs to input the datetime in UTC and specify a time zone. Then these lines are calculating what is the local time in such time zone based on the UTC time given. And then this local time is printed in the info methods. Although this is useful, I believe letting the user enter the local time is better than having to input time always in UTC.